src/modules/dashboard/DashboardEModeButton.tsx

```diff
@@ -82,23 +91,25 @@ export const DashboardEModeButton = ({
             alignItems: 'center',
             position: 'relative',
             zIndex: 1,
-            bgcolor: 'background.paper',
-            px: 1.5,
+            bgcolor: isEModeDisabled ? (open ? '#EAEBEF' : '#F7F7F9') : 'background.paper',
```

Comment:
For this, you could replace and rely on theme, which may be more readable:

`bgColor: isEModeDisabled ? (open ? theme.palette.divider : theme.palette.background.surface) : theme.palette.background.paper`

Reply:
With that said, I think we could possibly even add in a new background color for disabled: `theme.palette.background.disabled` that is the same light/dark values that the divider color uses